app/models/MetadataDefinition.scala

```diff
@@ -95,7 +95,12 @@ object MetadataDefinition {
         Json.parse("""
           {"label":"GeoJSON",
             "uri":"http://geojson.org/geojson-spec.html",
-            "type":"wkt"}""")
+            "type":"wkt"}"""),
+        Json.parse("""
```

Comment:
@lmarini instead of having this harcoded in the source, would it make sense to have this as json file that is loaded from the system (for example in the custom folder?)

Reply:
Totally, created new issue for it https://github.com/clowder-framework/clowder/issues/184